lib/lightcast-ruby/services/skills.rb

```diff
@@ -3,31 +3,40 @@
 module Lightcast
   module Services
     class Skills
-      def initialize(client:, version:)
+      def initialize(client:, version:, release:)
         @client   = client
         @version  = version
+        @release = release
       end
 
-      def extract(body = {}, query = { language: 'en', confidence_threshold: 0 })
+      def extract(**params)
+        # text: ''
+        # inputLocale: 'en-US'
+        # outputLocale: 'en-US'
+        # confidenceThreshold: 0.5
```

Comment:
@ericauv can you remove these. I don't want to have to worry about these coming out of sync with Lightcast's actuall API docs if they change these.